src/Plugin/Field/FieldFormatter/TextEDTFHumanFormatter.php

```diff
@@ -150,121 +182,140 @@ public function settingsSummary() {
    * {@inheritdoc}
    */
-   public function viewElements(FieldItemListInterface $items, $langcode) {
-     $element = array();
-     $entity = $items->getEntity();
-     $settings = $this->getSettings();
-
-     foreach ($items as $delta => $item) {
-       // Interval
-       list($begin, $end) = explode('/',$item->value);
+  public function viewElements(FieldItemListInterface $items, $langcode) {
+    $element = [];
 
-       $formatted_begin = $this->formatDate($begin);
+    foreach ($items as $delta => $item) {
+      // Interval.
+      list($begin, $end) = explode('/', $item->value);
 
-       // end either empty or valid extended interval values (5.2.3.)
-       if(empty($end)){
-         $element[$delta] = ['#markup' => $formatted_begin];
-       } elseif ($end === 'unknown' || $end === 'open') {
-         $element[$delta] = ['#markup' => $formatted_begin . ' ' .
-                                          t('to') . ' ' .
-                                          t($end)];
-       } else {
-         $formatted_end = $this->formatDate($end);
-         $element[$delta] = ['#markup' => $formatted_begin . ' ' .
-                                          t('to') . ' ' .
-                                          $formatted_end];
-       }
+      $formatted_begin = $this->formatDate($begin);
 
-     }
-     return $element;
-   }
+      // End either empty or valid extended interval values (5.2.3.)
+      if (empty($end)) {
+        $element[$delta] = ['#markup' => $formatted_begin];
+      }
+      elseif ($end === 'unknown' || $end === 'open') {
+        $element[$delta] = [
+          '#markup' => t('@begin to @end', [
+            '@begin' => $formatted_begin,
+            '@end' => $end,
+          ]),
+        ];
+      }
+      else {
+        $formatted_end = $this->formatDate($end);
+        $element[$delta] = [
+          '#markup' => t('@begin to @end', [
+            '@begin' => $formatted_begin,
+            '@end' => $formatted_end,
+          ]),
+        ];
+      }
 
-   protected function formatDate($edtf_text){
-     $settings = $this->getSettings();
-     $cleaned_datetime = $edtf_text;
-     // TODO: Time?
-
-     // Uncertainty
-     $qualifiers_format = "%s";
-     if(!(strpos($edtf_text, '~') === false)){
-       $qualifiers_format = t('approximately') . ' ' . $qualifiers_format;
-     }
-     if(!(strpos($edtf_text, '?') === false)){
-       $qualifiers_format .= ' (' . t('uncertain'.')');
-     }
-     $cleaned_datetime = str_replace(array('?','~'),'',$cleaned_datetime);
+    }
+    return $element;
+  }
 
-     list($year, $month, $day) = explode('-',$cleaned_datetime,3);
+  /**
+   * Create a date format string.
+   *
+   * @param string $edtf_text
+   *   The date to format.
+   *
+   * @return string
+   *   The date in EDTF format.
+   */
+  protected function formatDate($edtf_text) {
+    $settings = $this->getSettings();
+    $cleaned_datetime = $edtf_text;
+    // TODO: Time?
+    $qualifiers_format = '';
```

Comment:
This should be `$qualifiers_format = '%s';`. Otherwise, unqualified values (dates we are certain of) don't display at all.